library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/extras/focals/RectanglePromptFocal.java

```diff
@@ -150,6 +159,9 @@ public void prepare(@NonNull PromptOptions options, @NonNull View target, final
             mBaseBounds.bottom = top + height + mPadding;
             mBaseBoundsCentre.x = left + (width / 2);
             mBaseBoundsCentre.y = top + (height / 2);
+
+            mPath = new Path();
+            mPath.addRoundRect(mBaseBounds, mRx, mRy, Path.Direction.CW);
```

Comment:
Woops, a bit of an oversight by myself here, this should be in the update method. Now the prompt focus isn't pulsing :(